src/web/routes/application/check/get.js

```diff
@@ -47,13 +40,12 @@ const getCheck = (steps) => (req, res) => {
     claim: req.session.claim,
     ...pageContent({ translate: req.t }),
     checkRowData: getFlattenedRowData(req)(steps),
-    previous: getLastStepPath(steps)
+    previous: getPreviousPath(steps, steps[steps.length - 1], req.session)
```

Comment:
Is this going to work? If the last step's `isNavigable()` function returns true then it looks like this would return the path for the second last step, which would be incorrect.

Reply:
You;re right, it's only working now because we want to skip back over two steps.

Reply:
Updated to get the last step if navigable, otherwise call getPreviousPath